src/block-management/block-holder.js

```diff
@@ -189,6 +178,17 @@ export class BlockHolder extends React.Component<PropsType, StateType> {
 		);
 	}
 
+	getAccessibilityLabelForBlock() {
+		const { clientId, rootClientId } = this.props;
+		const order = this.props.getBlockIndex( clientId, rootClientId );
+		const name = this.props.getBlockName( clientId );
+		let blockType = getBlockType( name ).title;
+
+		blockType = blockType === 'Unrecognized Block' ? blockType : blockType + ' Block';
+
+		return `${ blockType }. Row ${ order + 1 }.`; // Use one indexing for better accessibility
```

Comment:
Correct me if I'm wrong but I believe we want internationalization for the accessibility label. we can use `sprintf` from `@wordpress/i18n` to add variables:
```js
let blockTitle = getBlockType( name ).title;
blockTitle = blockTitle === __( 'Unrecognized Block' ) ? blockTitle : sprintf( __( '%s Block' ), blockTitle );
return sprintf( __( '%1$s. Row %2$d.' ), blockTitle, order + 1 );
```

Reply:
> Correct me if I'm wrong but I believe we want internationalization for the accessibility label. 

Yep! You're right about this I'll push an update...

Hmm are you saying there's a sprintf function I can import like `import {sprintf} from '@wordpress/i18n'` or just `import sprintf from 'sprintf-js'`?

I take it for this to work I'll need to update the identifier in the tests as well to get that right but it's weird just changing it out locally doesn't work(which I'd expect it to since device language is English) or that's expected? 
Also is there a reason why `__( `${ blockTitle }. Row ${ order + 1 }.` );` wouldn't work?

Reply:
Yes! `import { sprintf } from '@wordpress/i18n';` would work.
`__( ${ blockTitle }. Row ${ order + 1 }. );` doesn't because xgettext (the tool that extract strings to be translated and creates POT files) need a string literal not a template for it to work.
So using `sprintf` is the recommended way for gutenberg to add variables to your strings.
See https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/block-mover/mover-description.js#L34 for an exemple of usage

Reply:
Ahh I see, thanks!